add clipBehaviour to Container

sahandevs opened the pull request:
## Description

added clipBehaviour to Container

```dart
if (clipBehaviour != null &&
        clipBehaviour != Clip.none &&
        decoration is BoxDecoration) {
      final BoxDecoration _decoration = decoration;
      current = ClipRRect(
        borderRadius: _decoration.borderRadius,
        clipBehavior: clipBehaviour,
        child: current,
      );
    }
```

i did cast the decoration to `BoxDecoration` because i need access to the borderRadius for the `ClipRRect`

## Related Issues

resolves #14421

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

<!-- Links -->
[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

Piinks commented on packages/flutter/lib/src/widgets/container.dart:
```diff
@@ -385,6 +386,10 @@ class Container extends StatelessWidget {
 
   /// The transformation matrix to apply before painting the container.
   final Matrix4 transform;
+  /// clipBehaviour for [BoxDecoration.borderRadius]
```
Nit: blank line above this, and a full sentence description
```suggestion
  
  /// clipBehaviour for [BoxDecoration.borderRadius]
```

Piinks reviewed (commented):
Hi @sahandevs,
Welcome! and thank you for the contribution! 🎉 
This change will need a test to ensure the behavior you are adding matches expectations.

sahandevs commented:
Hi @Piinks, Thanks!

I added tests and fixed the doc.

goderbauer commented on packages/flutter/lib/src/widgets/container.dart:
```diff
@@ -386,6 +387,14 @@ class Container extends StatelessWidget {
   final Matrix4 transform;
 
+  /// The clip behaviour when [BoxDecoration.borderRadius] is available.
+  /// 
```
nit: there's a trailing whitespace here that should be removed.

goderbauer commented on packages/flutter/lib/src/widgets/container.dart:
```diff
@@ -434,6 +443,17 @@ class Container extends StatelessWidget {
 
+    if (clipBehaviour != null &&
+        clipBehaviour != Clip.none &&
+        decoration is BoxDecoration) {
```
It's strange that the clipping will only work for BoxDecorations. What about a ShapeDecoration? Or any other decoration?

sahandevs replied:
we need to access the borderRadius inside the decoration
- We can just check for BoxDecoration ( Wich will support must use cases )
- Or We can create interface like ShapeWithBorder and add it to the all decorations
- Or we can just add another property in the Container as 'borderRadius' and use that

Wich one do you think is the best? @goderbauer

goderbauer replied:
We should ensure that this works for all decorations. We probably need to add something to the Decoration base class to get a clipPath (similar to ShapeBorder.getOutterPath) and then use that here.

Piinks commented:
Hi @sahandevs, are you planning on returning to this? It looks like the analyzer has detected whitespace in your changes.

sahandevs commented:
@Piinks I'm waiting for @goderbauer response

sahandevs commented:
@goderbauer I've added getBorderRadius to Decoration class and implemented it in BoxDecoration and ShapeDecoration if this approach is OK I will add docs and some tests for it

goderbauer commented on packages/flutter/lib/src/painting/shape_decoration.dart:
```diff
@@ -75,6 +76,7 @@ class ShapeDecoration extends Decoration {
     this.image,
     this.gradient,
     this.shadows,
+    this.borderRadius,
```
Adding a borderRadius here doesn't make much sense - especially since the class just seems to ignore it.

goderbauer replied:
Instead, a shapeDecoration should be clipped to its shape.

goderbauer commented on packages/flutter/lib/src/painting/decoration.dart:
```diff
@@ -165,6 +165,8 @@ abstract class Decoration extends Diagnosticable {
   /// if it is a [BoxDecoration] with definitely no [DecorationImage]).
   BoxPainter createBoxPainter([ VoidCallback onChanged ]);
+
+  BorderRadius getBorderRadius() => null;
```
Not all decorations have a border radius. This will probably have to return a more generic clip Path.

goderbauer commented:
Also: to fix the tests you'll have to merge the latest master into your PR.

sahandevs committed:
Revert "Convert to TextPosition for getWordBoundary (#44611)"

This reverts commit 36ae280795ab1bba451752fe78e2fc19bd701793.

sahandevs committed:
Revert "Re-land fix docker build and deploy_gallery-macos (#43654)"

This reverts commit f70214bdad819404eab94c7a49c3b8091191f9fd.

sahandevs committed:
Revert "pass --no-gen-bytecode to aot kernel compiler invocations (#43753)"

This reverts commit b07adb40556e6ed0e20d99053e23dad4a0cbe539.

sahandevs committed:
Revert "Sort locales and method/properties/getters alphabetically (#44761)"

This reverts commit 52a9f98ac9846b00ebb4ff8ce7f090dff54bbff8.

sahandevs committed:
Revert "Roll engine c448d39d5307..ddceed5f7af1 (2 commits) (#44380)"

This reverts commit d178b460907e8aba95c129d0ced6c9160517280b.

sahandevs committed:
Revert "Roll engine 49971e21402e..25563a12cab1 (7 commits) (#43695)"

This reverts commit 6cdf14dd9d2b16705a9a052228a989e183697baf.

sahandevs committed:
Revert "Catch MissingPortFile exception (#43573)"

This reverts commit ecf3e619de3d9e42dd780cd6847567cda5e85d8e.

sahandevs committed:
Revert "Roll engine 728e473f1a4b..c3b63d610b07 (2 commits) (#43661)"

This reverts commit 6ceb4d0146d1ff9e20bcffb022ca77d1b2a6117d.

sahandevs committed:
Revert "l10n tool improvements, stocks app refresh (#44313)"

This reverts commit db8676addcd43bc2c724ac27ceac6acaa02f226b.

sahandevs committed:
Revert "Add missing import (#43674)"

This reverts commit 2b44b925f44339691573f3ac2d9067a19bab2b82.

sahandevs committed:
Revert "Flip enable-android-embedding-v2 flag (#44369)"

This reverts commit 58adf2528a8512454f8cd0e938c710221e7adaea.

sahandevs committed:
Revert "Bump memory requirements for tool_tests-general-linux (#44620)"

This reverts commit f20867146320b76476f78a7a998faa28d836e5c6.

sahandevs committed:
Revert "flutter build aar should also build plugins as AARs (#43994)"

This reverts commit 41e1dd2f919834544db1222387ab7268cedf24d1.

sahandevs committed:
Revert "Exit tool if a plugin supports the embedding v2 but the app doesn't (#44026)"

This reverts commit ae49775dce60775ac49f11c8ba0d7915e199a015.

sahandevs committed:
Revert "Fix stdout test (#43927)"

This reverts commit 3e0d6ea8c549bd70eb411d4bcfc0cc07e4cde999.

sahandevs committed:
Revert "Roll engine e73c9c8f6bd5..7b968ff95ff1 (3 commits) (#44158)"

This reverts commit d770c12b323028aac34753578a26f455a111289f.

sahandevs committed:
Revert "Roll engine e609577d1201..97807b534122 (3 commits) (#43860)"

This reverts commit 667aebb7eae137aff7f0ecc7d4457e9ab47877e0.

sahandevs committed:
Revert "Separate DropdownButton and DropdownButtonFormField tests (#43738)"

This reverts commit 3d52ee54f449d94dd24e5a9ae03aaabdbfc3400e.

sahandevs committed:
Revert "Roll engine 6a1f093398c6..5f5713e33971 (5 commits) (#44448)"

This reverts commit 1811a69cf4679f7e77b5261b12e4d21c68834ce6.

sahandevs committed:
Revert "Improved ios 13 scrollbar fidelity (#41799)"

This reverts commit affd6551cd0fc0992df7b46ac4febe820291c5cc.

sahandevs committed:
Revert "Roll engine 5f5713e33971..af04338413c3 (8 commits) (#44486)"

This reverts commit e1114987f5a51f38a108be465ffe0f5a9b7239a3.

sahandevs committed:
Revert "Deprecation cleanup in flutter_tools (#44469)"

This reverts commit 4e2da7c10c90d35e06474ae3fd450a3bf98c354a.

sahandevs committed:
Revert "abaa85a39 [web] Proper support for text field's obscureText (flutter/engine#13722) (#44311)"

This reverts commit dae0cd90715e62f386634e1ecf35acc40fbd3677.

sahandevs committed:
Revert "Don't add x86 nor x64 when building a local engine in debug mode (#44302)"

This reverts commit 73b1923d8b10e910337c2495ef274d859214210d.

sahandevs committed:
Revert "Reland: Migrate examples new embedding (#43868)"

This reverts commit 0ec19773f050f4e50fb31f680d7dea2ecc10b72e.

sahandevs committed:
Revert "Revert: Migrate examples to Android embedding v2 (#43997)"

This reverts commit 25a07ec923f6a6b01ad3a5668ae66e987486a036.

sahandevs committed:
Revert "Enable usage of experimental incremental compiler for web (#43576)"

This reverts commit c5acb09cd6ae03e3ceb2448b79576a4d2822221e.

sahandevs committed:
Revert "Refactor BuildMode into class, add jit_release configuration (#42476)"

This reverts commit e2b7aba5d18fb428ec2984c052506eb593bf6a63.

sahandevs committed:
Revert "Re-Land: Add focus nodes, hover, and shortcuts to switches, checkboxes, and radio buttons. (#43657)"

This reverts commit 25b38ea5d96ecf6c14a1269d367c6856d2c88dd7.

sahandevs committed:
Revert "Deflake flutter tool logger test (#43800)"

This reverts commit 713129be83e04778726c69db192a667268bbf76f.

sahandevs committed:
Revert "Roll engine 869b74eb4eca..728e473f1a4b (2 commits) (#43652)"

This reverts commit 8708598ce95dbc4b1d5e79ac82586f40c755f343.

sahandevs committed:
Revert "Roll engine 46c757be07fd..36303c61b63f (3 commits) (#43784)"

This reverts commit be10a1e4f4a4adc58db2d4756ef1a1cb8835e625.

sahandevs committed:
Revert "Adding subtitle to ExpansionTile (#43946)"

This reverts commit c9b01573ebb6b151ad62b4ce2207c44f0e23f7ae.

sahandevs committed:
Revert "Re-enable Pesto in profile/release mode (#44299)"

This reverts commit e7a51d5fa34975776b71cc5525cdb0f3cced89ea.

sahandevs committed:
Revert "3fb5d855f Roll fuchsia/sdk/core/mac-amd64 from xC_an... to 4JJNo... (flutter/engine#13773) (#44603)"

This reverts commit c34b3bea70300c687042eb18307c4e092ba1c7cd.

sahandevs committed:
Revert "[flutter_tool] --flutter_runner will download the debug symbols (#44119)"

This reverts commit 908c7ca74c81631fac1fb35c8571124f74e8f24d.

sahandevs committed:
Revert "l10n Localizations and LocalizationsDelegate Generation Tool (#43934)"

This reverts commit d4550dceec2cb56a0c58f551d80d107d89d7c2d2.

sahandevs committed:
Revert "3ebf00690 Deprecated DartExecutor as BinaryMessenger and added a getBinaryMessenger() method. (#43202) (flutter/engine#13349) (#43562)"

This reverts commit 215284f2bf42e309f4809a661a4dd7a68f8ba91d.

sahandevs committed:
Revert "Re-Land: Add focus nodes, hover, and shortcuts to switches, checkboxes, and radio buttons. (#43384)"

This reverts commit cc6f7e8df4afdebcdbf643d92c15bc317b9bcc2b.

sahandevs committed:
Revert "Roll engine 2f69e38160a3..b4a50d7ab192 (2 commits) (#44497)"

This reverts commit 1c21850e33bffe5aa4846d7e8bf8d3ce0dc8f5d3.

sahandevs committed:
Revert "Fixed usage of optional types in swift. (#43461)"

This reverts commit bbd8302fa1eac0548fdc11c6c3ff36309f4c28a0.

sahandevs committed:
Revert "Turn off docs upload temporarily (#44365) (#44413)"

This reverts commit 327a7a1fde34a56ef944082567132d6f7983069a.

sahandevs committed:
Revert "Update packages --force-upgrade (#43745)"

This reverts commit 211f640b9640880c3540d81cc2d81ff637294590.

sahandevs committed:
Revert "44c9fab7b Roll src/third_party/dart d50c158d42..bbe2ac28c9 (19 commits) (#43937)"

This reverts commit e55efa8951e1d379bfcbfb03951098b55eef7912.

sahandevs committed:
Revert "Do not pass obsolete --strong option to front-end server (#44278)"

This reverts commit 54e7b5ce0126346a390fc98b915b7c3e88b15003.

sahandevs committed:
Revert "add --dart-defines option (#44083)"

This reverts commit e08a3104a48d03c4b84ea56a273d7e092de664b1.

sahandevs committed:
Revert "Revert "Migrate examples to the Android embedding v2 (#43221)" (#43611)"

This reverts commit 15d84167ceb26b29a8709ac9ec09684068d3464b.

sahandevs committed:
Revert "Re-enable docs uploading (#44454)"

This reverts commit af0113ed8ca2dece27c50222d044b95f6d533409.

sahandevs committed:
Revert "Manually roll engine to unred the tree (#44150)"

This reverts commit 51ea813ca45c06155e7b707a49a76d2a5b8c6bd5.

sahandevs committed:
Revert "fixing todo comments in flutter tests to include url link. (#42691)"

This reverts commit c43cf997762e36f05bf22b96087f4f6ae8c21496.

sahandevs committed:
Revert "Update packages to get latest dwds (#44344)"

This reverts commit 7da2a5ab47f38b8c97c980adc89c770c2e4c6906.

sahandevs committed:
Revert "Added tests for the new Android heading semantic flag to android_semantics_testing (#44031)"

This reverts commit 7a2ea7203676571ec0bc94cfb44895d491fbe0c5.

sahandevs committed:
Revert "31cd2dfca Remove usage of fuchsia.modular.Clipboard. (flutter/engine#13763) (#44503)"

This reverts commit 1e9a18b48b4c86f4af7e1cce670ff347b561189e.

sahandevs committed:
Revert "4b76fadbf Fizzle onConfigurationChanged if no FlutterView (flutter/engine#13445) (#43867)"

This reverts commit 51cc4c3fb733f516d6cf322b5b8698fa8e9a5161.

sahandevs committed:
Revert "Make module_test_ios pass (#44147)"

This reverts commit 654b71a9813c2dd0e24a79a39883eed6fc5ae6d4.

sahandevs committed:
Revert "Make ProjectFileInvalidator injectable (#44200)"

This reverts commit e8a97af2ebed870a37a1a5f2f6ca4f84660b11b2.

sahandevs committed:
Revert "Adjust and refactor all RaisedButton tests into its respective file (#44169)"

This reverts commit 5e7aae2d281dd704faa45357749b8f618c13a212.

sahandevs committed:
Revert "Canonicalize locale string in `gen_l10n.dart` (#44758)"

This reverts commit 464a5670ea6b81007271008d5e1806e23be9cc58.

sahandevs committed:
Revert "CupertinoSlidingSegmentedControl (#42775)"

This reverts commit b01613d35245b8b0ffa4f1d2e90f9e6e1b059a87.

sahandevs committed:
Revert "[flutter_tool] Use engine flutter_runner prebuilts (#43381)"

This reverts commit 6ed30cf5271307a98e4742564a1fc164a6de0ba8.

sahandevs committed:
Revert "Re-enable chrome dev mode tests (#43691)"

This reverts commit 09e5849d80c8780ef2c0f233206e38fe7cfb9027.

sahandevs committed:
Revert "Support platform-specific test lines (#43458)"

This reverts commit 994ca76c507ca3e2e23a8f7e8906ea6dbd6f6ef1.

sahandevs committed:
Revert "Fix AboutListTile having an empty icon placeholder when no icon set. (#43422)"

This reverts commit 7355fa61d3b8a41b4b21d9787cbb8032cb521a2f.

sahandevs committed:
Revert "Enable tests that failed due to CupertinoDynamicColor (#43636)"

This reverts commit 5a146d20478f6f00842d971048259ecf072d0905.

sahandevs committed:
Revert "Roll engine 44c9fab7b4d8..8ea19b1c76f6 (4 commits) (#43942)"

This reverts commit 9210fe9ec84433f4256e7594e5ccc042d401b925.

sahandevs committed:
Revert "Handle onError callback with optional argument (#43436)"

This reverts commit 04995d67a1078703f2f36db0336dcc5094998df7.

sahandevs committed:
Revert "629930e88 Prefer SchedulerBinding.addTimingsCallback (flutter/engine#13728) (#44388)"

This reverts commit 164b93e571099d809a860fe0b08bd24211400092.

sahandevs committed:
Revert "Adjust and refactor all `MaterialButton` tests into its respective file (#43528)"

This reverts commit 87cd2295bfddb57a7a7a06d6b34d02dcac70fe75.

sahandevs committed:
Revert "Revert "Enable usage of experimental incremental compiler for web (#43576)" (#44396)"

This reverts commit 3fd4f61b4a690e6cea3578a315202ddbe52188f4.

sahandevs committed:
Revert "Update CupertinoSlidingSegmentedControl control/feedback mechanism (#43932)"

This reverts commit 41dd279de524e2617604aca99318c0d01e53d3b9.

sahandevs committed:
Revert "Apply minimumDate & maximumDate constraints in CupertinoDatePicker date mode  (#44149)"

This reverts commit 848a1260f579a5caca05bb505358a17cdaa8967a.

sahandevs committed:
Revert "Added a null check for ranges in the sourceReport map. (#43667)"

This reverts commit 900201454757c43bdd9ba4ebe4f40b40af7fbb72.

sahandevs committed:
Revert "[flutter_tool] Screenshot command must require device only for _kDeviceType (#44227)"

This reverts commit aad020e99554ffe9c1230646d6678de2df3c0d46.

sahandevs committed:
Revert "Upgrade dartdoc to 0.29.0 (#43990)"

This reverts commit 021b80dbd393658d359ff0151c0791b658d40163.

sahandevs committed:
Revert "[flutter_runner] Use sky_engine from the topaz tree (#43684)"

This reverts commit f54d2d7cd1041483f0206e2f87b3b10326baea6a.

sahandevs committed:
Revert "Fix typo in app_bar.dart comment (#43981)"

This reverts commit a8ddb40fb89050dd270ec490e34ec65849417ad4.

sahandevs committed:
Revert "SliverAppBar - Configurable overscroll stretch with callback feature & FlexibleSpaceBar support (#42250)"

This reverts commit bcc89857789460bdaa3780c0bc1ec799a8875db0.

sahandevs committed:
Revert "419f5d594 Roll src/third_party/skia 1beb8ae9a2f5..f22dda960136 (11 commits) (flutter/engine#13409) (#43698)"

This reverts commit 561d30faccf07b92df0cb678f550e77a304c3ff7.

sahandevs committed:
Revert "Adjust and refactor all OutlineButton tests into its respective file (#44328)"

This reverts commit 5f8847fa493070428053b2490123db6722fd0d41.

sahandevs committed:
Revert "Update README to include non-mobile targets (#42873)"

This reverts commit ffbdd19eb27844540ce211e1d3ab715bbba5ba64.

sahandevs committed:
Revert "d68c00947 Roll fuchsia/sdk/core/mac-amd64 from 6Du8E... to r1tS4... (flutter/engine#13726) (#44319)"

This reverts commit 62dd21e10ebec54cc57febb4a15fa72a874ab421.

sahandevs committed:
Revert "Roll engine 4b76fadbf438..7c80d62e89f6 (5 commits) (#43920)"

This reverts commit ee42bf0eae27482eb18dd4009ec67537a7030a1c.

sahandevs committed:
Revert "Revert "Demonstrate that artifact invalidation works (#44312)" (#44463)"

This reverts commit 74279ceb8fcfba2482e8f5bb4537f27f87334f67.

sahandevs committed:
Revert "Refactor flutter.gradle to use assemble directly (#43876)"

This reverts commit e096d3fc6276339fbe50acc824e88e5f845d7767.

sahandevs committed:
Revert "Make sure all our .dart files have license headers (#44467)"

This reverts commit a20df069d863efa6f75323947b3cfa92a8fbb2fa.

sahandevs committed:
Revert "[ci] Use the latest Cirrus Image for macOS  (#44576)"

This reverts commit 384da775971c57c2de8270c16f560f661d8f04f3.

sahandevs committed:
Revert "Allow specifying device-vmservice-port and host-vmservice-port (#44027)"

This reverts commit 2bbb85d8f58864c94fa3bf25e7c64fbf7f555d7f.

sahandevs committed:
Revert "Fix comment typo in tabs.dart (#44068)"

This reverts commit 6b0629246845d00506870336df77862e586f810b.

sahandevs committed:
Revert "implement build aot with assemble for Android target platforms (#43282)"

This reverts commit beeb9bb3f74e35363ac0121828d9c6d6f84fc1e0.

sahandevs committed:
Revert "added new lifecycle state (#39945)"

This reverts commit bbb69ae7e2e04311cf88933a7fd7675bd0d82658.

sahandevs committed:
Revert "Revert engine rolls. (#44613)"

This reverts commit 8b493f3bebe6bbcf7b73176587fa61b4b6cbaac9.

sahandevs committed:
Revert "Reland: enable usage of experimental web compiler (#44400)"

This reverts commit aca5b234639f4221d4c62c893b39d8281b01fe81.

sahandevs committed:
Revert "Forward ProcessException to error handlers (#44783)"

This reverts commit 451a0fedcdb40e9fdc426d1b5aa6d7d1e9d6e053.

sahandevs committed:
Revert "[flutter_tool] Update analytics policy, send event on disable (#43217)"

This reverts commit 26b72bbff3370905430ef1514b6499378a430b08.

sahandevs committed:
Revert "Don't print how to consume AARs when building plugins as AARs (#44301)"

This reverts commit cfbc097b9092a016d955892e55083b63d3ee0ee4.

sahandevs committed:
Revert "Use PRODUCT_BUNDLE_IDENTIFIER from buildSettings to find correct bundle id on iOS when using flavors (#42964)"

This reverts commit b19e77db9cdf8387ab411dbe7aed8d1a7a38c9d1.

sahandevs committed:
Revert "copy chrome preferences to seeded data dir (#44032)"

This reverts commit d1f87867dac0d835a67cc488274528b78719ca56.

sahandevs committed:
Revert "Synchronize modifier keys in RawKeyboard.keysPressed with modifier flags on events. (#43948)"

This reverts commit d12454f11978706d29ce8613d0b900cf57ddcd2a.

sahandevs committed:
Revert "Typo in TextFormField comments (#44076)"

This reverts commit 0d34d59677e91e9694601e085e7c2ef20c261b4e.

sahandevs committed:
Revert "Update Stocks example using i18n tool  (#44223)"

This reverts commit c716724897758536e63626140e928cad5be6d28b.

sahandevs committed:
Revert "Gold Performance improvements (#43748)"

This reverts commit f0579e3dd8488475c87615764994a1d42b668c52.

sahandevs committed:
Revert "Sort Localization generation output (#44743)"

This reverts commit 8749ddf01e699e8812987115bc15c73700422752.

sahandevs committed:
Revert "Always link desktop platforms (#44753)"

This reverts commit 1b84d8a3ec65a5ffdafab5ef82c4098535eb8936.

sahandevs committed:
Revert "Roll engine abaa85a394c8..34a40cfbfd36 (2 commits) (#44315)"

This reverts commit 38c8d1f0cb5197a686a94c84271a054ea96fcab3.

sahandevs committed:
Revert "Add "navigator" option to "showDialog" and "showGeneralDialog" (#42842)"

This reverts commit a1362f4d3449e9ab9a061bd3e82b0decb85aace2.

sahandevs committed:
Revert "set trace to true for desktop builds (#43577)"

This reverts commit 25d6bd11c5629b32a8547c9a899503851a803bb0.

sahandevs committed:
Revert "Build ios framework (#44065)"

This reverts commit 99952c99e59bdc4dfa85fb66b7081f0d59fc4bf7.

sahandevs committed:
Revert "Add onLongPress to Buttons (#40641)"

This reverts commit 587f5488966239a4c943211828c919eac24beb82.

sahandevs committed:
Revert "Adding handling of TextInputClient.onConnectionClosed messages handli… (#43466)"

This reverts commit 29010d48ec28ab28d4f3d7597252e4b30d2f0dd2.

sahandevs committed:
Revert "Ignore generated .project files for VSCode Java Plugin (#44019)"

This reverts commit a25a1b50cd912b0f94189f58fbe85df2564b8276.

sahandevs committed:
Revert "revert add new enum change (#44281)"

This reverts commit 263c90b74e52cd0a0011ff6368e0346b84072c56.

sahandevs committed:
Revert "Increase TextField's minimum height from 40 to 48 (#42449)"

This reverts commit 07dd7f81ce43592d68cffa3b2b59fbceaf4a973f.

sahandevs committed:
Revert "Revert "Turn off docs upload temporarily (#44365)" (#44371)"

This reverts commit 579740c12a2849cff38982c71879c4c7966f9712.

sahandevs committed:
Revert "Remove flutter.yaml migration code (#44146)"

This reverts commit 9edb5c235e459bef59657df36c08ef1548f6e5f7.

sahandevs committed:
Revert "Update cupertino demos in gallery (#43841)"

This reverts commit e25ba052430f76237b1cd42cc13922f57934f45f.

sahandevs committed:
Revert "switch web test to macOS (#44421)"

This reverts commit 4860e82aaa81f22b3e95c6222c05ba471c2eda63.

sahandevs committed:
Revert "Add enableSuggestions flag to TextField and TextFormField (#42550)"

This reverts commit 6c5bcb81d4fefd306f4cd25e6ea0a249b58f8cce.

sahandevs committed:
Revert "Change PopupMenuButton.icon type to Widget (#43526)"

This reverts commit 6ad6a8c65914a1ba9aaade1eecacf9eefad91fa0.

sahandevs committed:
Revert "c448d39d5 Roll src/third_party/skia 714f8cc3ff4b..f66ec5d69d04 (3 commits) (flutter/engine#13729) (#44321)"

This reverts commit ba3442804a511eea31a509a3b0627403b6c5c0df.

sahandevs committed:
Revert "CupertinoDynamicColor improvements (#44317)"

This reverts commit 0e0ccb2f477e8665e0b1a8c24af2e955924a713b.

sahandevs committed:
Revert "Build local maven repo when using local engine (#44243)"

This reverts commit 636f2dd93ea602fbfd3bafe1371a9157c813d2c1.

sahandevs committed:
Revert "Serve correct mime type on release dev server (#43907)"

This reverts commit d1fddae7a1d90af7fb82f7be85685d630e52571a.

sahandevs committed:
Revert "Roll engine 8ebb31840134..2a8f6dca6006 (2 commits) (#44303)"

This reverts commit 4718effd432ff7c28552fbe2dc545f692be354bf.

sahandevs committed:
Revert "Alias upgrade-packages => update-packages (#43546)"

This reverts commit 40359dfb158fdbaffab4b9276f2cb76c1ec1063c.

sahandevs committed:
Revert "No multiline password fields (#42470)"

This reverts commit 08e434ce97ead7c798f0fccbee9980dca9c32090.

sahandevs committed:
Revert "CupertinoPicker doc update (#43736)"

This reverts commit 08ed4b8c8164ca6069c97a90bc207f15abe387e2.

sahandevs committed:
Revert "Roll engine 7b968ff95ff1..0928c1af2f45 (2 commits) (#44215)"

This reverts commit b8052539064edd09f2baf91035e40693df056d1f.

sahandevs committed:
Revert "Wire selectedItemBuilder through DropdownButtonFormField (#44160)"

This reverts commit 0e2874640738f017af3b31c77a005363d7519e80.

sahandevs committed:
Revert "Remove Source.behavior, fix bug in depfile invalidation (#43945)"

This reverts commit af6c8095ea2c63ba0b0ad48e3c6599303e27ecdd.

sahandevs committed:
Revert "Roll engine 419f5d594afc..46c757be07fd (6 commits) (#43769)"

This reverts commit 70a520a80e59c95cfdf185c03c6db30f28fa0d66.

sahandevs committed:
Revert "Make Spinner test case use mockStopwatch (#43761)"

This reverts commit 3359b4861178e906f7c422d2b7c142843e0d1522.

sahandevs committed:
Revert "Add Android embedding version analytics (#44043)"

This reverts commit 920fb9576f45c910c6bee82a868133d445c1032f.

sahandevs committed:
Revert "Added note about design doc template. (#43658)"

This reverts commit 2ba808d90327dde601bc2c992d8ea8c37add2f85.

sahandevs committed:
Revert "CupertinoContextMenu (iOS 13) (#43918)"

This reverts commit 1467c50bdb08b1bf48fdccf1c75ec834dd2039bb.

sahandevs committed:
Revert "b16eab6af Remove usage of yaml module from CIPD script (flutter/engine#13444) (#43864)"

This reverts commit 2cf71854c41bf34f5a64ca071c1a564e4a9bdc80.

sahandevs committed:
Revert "ModalBarrier and Drawer barrier prevents mouse events (#44296)"

This reverts commit f9ade49f85407f2175910f08834f1388c3df0c08.

sahandevs committed:
Revert "Turn on bitcode for integration tests and add-to-app templates (#44633)"

This reverts commit e5b66a90ec73696f4f7599a07c7e724e9bb5763c.

sahandevs committed:
Revert "remove yield from inherited model (#44233)"

This reverts commit 903757ea7f4a958ce7cceeeee3544a5465bffe7e.

sahandevs committed:
Revert "Tap.dart: Fixes the spacing to the right side of reason (#43246)"

This reverts commit 9e684651ada16fdc2c361d64ce40e490d5942491.

sahandevs committed:
Revert "Turn off docs upload temporarily (#44365)"

This reverts commit 52e958dacedb413cebada04e9b9acd33dfbdac48.

sahandevs committed:
Revert "Show a warning when a module uses a v1 only plugin (#44499)"

This reverts commit e3b3c1e710f8d6300639cce0f0dcc59ceb4c9ea1.

sahandevs committed:
Revert "Don't indefinitely persist file hashes, handle more error conditions (#43602)"

This reverts commit 36a85cde23fe57e6e2569b669e72f756f3ebf999.

sahandevs committed:
Revert "Add more structure to errors (continuation of #34684) (#42640)"

This reverts commit bb0bfe1df2ef6d4fbeb52c5679e91a8de5fd823c.

sahandevs committed:
Revert "l10n tool improvements, stocks app i18n refresh (#44473)"

This reverts commit 9e40bd7eb71489da0dd3e9a4285854c45e80b3a8.

sahandevs committed:
Revert "Use raw string for l10n description  (#44451)"

This reverts commit 7a623b19e31401ae0c6c7604efccd1b6ae7bc54b.

sahandevs committed:
Revert "Revert "[flutter_runner] Use sky_engine from the topaz tree (#43684)" (#43913)"

This reverts commit 97affffdcea8d51206e703a558e408f0c04df9d1.

sahandevs committed:
Revert "Catch failed startup error from build_daemon (#43598)"

This reverts commit 3f45b0be0b2b53d7c9c6996052eb7b97947b5690.

sahandevs committed:
Revert "Reland "Enable dump-skp-on-shader-compilation in drive (#43022)" (#43455)"

This reverts commit e1602980eaea94a0aed023d63cf523bfdbaa1067.

sahandevs committed:
Revert "Add more flutter build ios-framework tests before the impending jonahpocalypse (#44308)"

This reverts commit 53ebf641f7482965f2b3cf1c694fca234e32b10f.

sahandevs committed:
Revert "fix test for null flutter root (#44468)"

This reverts commit 94b067bcf357bf3aa8d98226eee3b9a78d2d5050.

sahandevs committed:
Revert "Revert "Roll engine 6c763bb551cb..9726b4cb99d3 (4 commits) (#44251)" (#44277)"

This reverts commit 099dad0c61c5243ca09c4fe666c3836444f69117.

sahandevs committed:
Revert "Roll engine af04338413c3..2f69e38160a3 (2 commits) (#44493)"

This reverts commit 1cecad1f6cd0c449a24298531a4e9c3b1e25aba3.

sahandevs committed:
Revert "Roll engine 05ab04dbe8cf..6c763bb551cb (3 commits) (#44241)"

This reverts commit 66a36a94087d984b773e75a55af4ec9cc790a517.

sahandevs committed:
Revert "Roll engine 31cd2dfca22a..eec36a8d4f6d (6 commits) (#44533)"

This reverts commit 3aedd72aba4a7fe0f6de30e09f7820bacef583c1.

sahandevs committed:
Revert "Make selected item get focus when dropdown is opened (#43722)"

This reverts commit 192476482451620b6b5e42d09287090c6cc60705.

sahandevs committed:
Revert "Use platform appropriate filepaths when constructing asset bundle (#44221)"

This reverts commit 0fea92e170b504f14326c8c35894b83b9a28e212.

sahandevs committed:
Revert "Remove Poller class from flutter_tools (#43685)"

This reverts commit 465e1710f5c124bb1cbeea00134e3159ac3a4130.

sahandevs committed:
Revert "Split desktop config fallback variable by platform (#43758)"

This reverts commit 831ceb2b7c2a89e76226ba3c46af30a81c616428.

sahandevs committed:
Revert "support --no-resident on the web (#44028)"

This reverts commit 7d99c89d38253b8f0921d059c00b0fbf6c6e9c72.

sahandevs committed:
Revert "Roll engine 629930e8887c..7ed9c9fc5885 (2 commits) (#44401)"

This reverts commit 4321488cdb44fcfcf3848ec7520b1b72696386f2.

sahandevs committed:
Revert "Revert "Reland: Migrate examples new embedding (#43868)" (#43885)"

This reverts commit ca4989964bcf5283b4374c46e34eb10710f6ce71.

sahandevs committed:
Revert "build aar prints how to consume the artifacts (#44127)"

This reverts commit 0658a3d2ba397367a3cb1734680524c6af80bef6.

sahandevs committed:
Revert "Ensure web-server does not force usage of dwds (#44744)"

This reverts commit 2f8bed219bc8e8e636587c8f470296ddee28ee14.

sahandevs committed:
Revert "Demonstrate that artifact invalidation works (#44312)"

This reverts commit f20a485e6b9ad1ca44e379ce52c595defbc9f1b6.

sahandevs committed:
Revert "Update meta to 1.1.8 (#44584)"

This reverts commit 4835e63785362da33f9a061fd49921d61cc50c7b.

sahandevs committed:
Revert "Observe logging from VM service on iOS 13 (#43915)"

This reverts commit 0916dc855982f6bed7277cec67a31316a477b0b9.

sahandevs committed:
Revert "Reorder show and setEditingState calls to the IMM (#43865)"

This reverts commit 70f410099d236c7e52bddea8735ca290c224dd6f.

sahandevs committed:
Revert "Add swift_versions to plugin template podspec, include default CocoaPod version (#44324)"

This reverts commit c405e6fc4b19359bc27133c4634fee7941917b63.

sahandevs committed:
Revert "Allow multiple TimingsCallbacks (#43676)"

This reverts commit 326df0a0a046cf82e510f19d6dbb79e083f25843.

sahandevs committed:
Revert "Respond to TextInputClient.reattach messages. (#43959)"

This reverts commit c6f06c319cec8019ea0265bfc84305b4dc4af9f3.

sahandevs committed:
Revert "make more spinner tests not flaky (#43955)"

This reverts commit 7e90fe70d783e1d3637a6a13cfaa0611550c8fb6.

sahandevs committed:
Revert "Refactor gradle.dart (#43479)"

This reverts commit a9b7dc8bf8b8094fe2c429a66b7bdee4ec7b0f9b.

sahandevs committed:
Revert "Revert "Added a null check for ranges in the sourceReport map. (#43667)" (#43827)"

This reverts commit acb1095247fbaf5373fbc9deb855924021640f4d.

sahandevs committed:
Revert "Log additional Android build failures (#43941)"

This reverts commit e38153ceee3532deee7ee884b51328569587282f.

sahandevs committed:
Revert "Pad CupertinoAlertDialog with MediaQuery viewInsets (#42967)"

This reverts commit ecc39aa59df05eab1b53342480df7ce507c792da.

sahandevs committed:
Revert "Remove new unused elements (#44551)"

This reverts commit 34fde1df91ca36547bc6ef991683be6c04061581.

sahandevs committed:
Revert "add libzip cache artifact (#43677)"

This reverts commit f0733ecdcc807d7d422f8093a80d16587b76c25c.

sahandevs committed:
Revert "Implement AlertDialog title/content overflow scroll (#43226)"

This reverts commit f21562fc4cea9738f1042411f69d1985f6a64852.

sahandevs committed:
Revert "Revert "Exit tool if a plugin supports the embedding v2 but the app doesn't (#44026)""

This reverts commit 00ec1920aa79094a1139faf7dc0bcb95ac52b4d0.

sahandevs committed:
Revert "Fix v1 embedding support heuristic for plugins (#44214)"

This reverts commit 536033fdbb6b5325d3d0472bc33e7ca462438a31.

sahandevs committed:
Revert "Add convenience accessor for primaryFocus (#43859)"

This reverts commit 2a9825ff44accb5c5adfe5291d023f4bdc48211a.

sahandevs committed:
Revert "refactorings to testbed.run and testbed.test (#44488)"

This reverts commit ed05f01a1ad17f165a8cd20d54618a6a903faa85.

sahandevs committed:
Revert "Update dartdoc to 0.29.1 (#44466)"

This reverts commit bd8b306457356a88a9918bf0001ad991c496d2d4.

sahandevs committed:
Revert "Manual roll for add new enum roll back (#44292)"

This reverts commit ce3308a966fcaf57df353145825323d0affcc927.

sahandevs committed:
Revert "Remove TODO that's done (#44318)"

This reverts commit 97189cb423b6748f8d20af9ec2f5c61d6d861686.

sahandevs committed:
Revert "Temporarily bring the embedding dependencies in the Flutter gallery (#44201)"

This reverts commit 99a245563f4842db017dfeae281b0f2365b56e03.

sahandevs committed:
Revert "Flutter doctor should require java 1.8+ (#41989)"

This reverts commit 3a14d3c0d8549fca731b6cc03a10367cd1313fa7.

sahandevs committed:
Revert "Improve DropdownButton assert message (#43511)"

This reverts commit 6cd5a6c22dde3575e1f9a891b645683735404cb7.

sahandevs committed:
Revert "link platform should be true for profile (#44279)"

This reverts commit ef3727a0efa0a07d551ae130c69cec9bb30feee0.

sahandevs committed:
Revert "Roll engine 6c257df8e7b9..869b74eb4eca (15 commits) (#43637)"

This reverts commit ec12f4ca60592e97619d841d20a5509d519772e2.

sahandevs committed:
Revert "Reland: Migrate examples new embedding (#43643)"

This reverts commit b5e7d6a2b986de51ce4ce93f9b10f532763603ae.

sahandevs committed:
Revert "Remove print and fix code formatting (#43843)"

This reverts commit e706a99f4735d3dfd212acbada02d0595d3d4ff8.

sahandevs committed:
Revert "[flutter_tool] Stream artifact downloads to files (#44360)"

This reverts commit c573bbb16ddb098882d9f8f833431694201871a4.

sahandevs committed:
Revert "Mark routes as opaque when added without animation (#43756)"

This reverts commit af1d4e638f9944fa6d45318efc741ca8f9c62458.

sahandevs committed:
Revert "Roll engine 0a8bd9dd6fbb..e73c9c8f6bd5 (13 commits) (#44156)"

This reverts commit 8718e5f3c72eac322e64a531ae8d0d91fd98cb26.

sahandevs committed:
Revert "fix text tests on the web (#44021)"

This reverts commit a43e13bdbf0cc11bbccb6a944e14c8adedd67dec.

sahandevs committed:
Revert "Remove TextRange, export it from dart:ui (#44422)"

This reverts commit b8da337f1b77b0108d96f47a14d7782a112a71b9.

sahandevs committed:
Revert "Pass environment variables through to xcodebuild (#43553)"

This reverts commit ecf9280b0fb316912da0351054d86b1a9400eadb.

sahandevs committed:
Revert "Use alphabetic baselines for layout of InputDecorator (#44029)"

This reverts commit 95c9092ffbd3cb2a9733fa3f7a4949cb836ac5bc.

sahandevs committed:
Revert "Fix device lab tests (#43675)"

This reverts commit f7a16d8c0bc23b881f2f7774337d3f05585000bc.

sahandevs committed:
Revert "remove services code (#44052)"

This reverts commit 8c3fd5a3c18b4b459f5452220b9e769574e6e7e9.

sahandevs committed:
Revert "Adjust and refactor all `FlatButton` tests into its respective file (#43742)"

This reverts commit efd580a54419ce3a757765212929b19f4667b250.

sahandevs committed:
Revert "Check in new diffs to material localizations (#44736)"

This reverts commit a7eaf4dc46cc0905efb98af2bc218fef67405bf8.

sahandevs committed:
Revert "Don't read AndroidManifest.xml if it doesn't exit (#43669)"

This reverts commit ae8d842f026cd003d8b739560a4126a144b7dd39.

sahandevs committed:
Revert "Migrate examples to the Android embedding v2 (#43221)"

This reverts commit 5156a80a89e8649d627d17c4785e0acd4624417f.

sahandevs committed:
Revert "Revert "Implement AlertDialog title/content overflow scroll #43226" (#44003)"

This reverts commit 079dc543a9c66bff6f8f2f9fcdef2ab6a63bd4cc.

sahandevs committed:
Revert "Roll engine 36303c61b63f..5051bef17bdc (21 commits) (#43828)"

This reverts commit 961f033d86ea2e3580ef5ddf0785de57a33d95c5.

sahandevs committed:
Revert "Enable heroes_test.dart on the web matrix (#43662)"

This reverts commit 19d5856aef042e219e5c5a55e684886d69ce75ee.

sahandevs committed:
Revert "require awaiting Testbed.run (#43952)"

This reverts commit 484d0a2c47897d15f8fa466c8e8df647c7785643.

sahandevs committed:
Revert "Revert "Re-Land: Add focus nodes, hover, and shortcuts to switches, checkboxes, and radio buttons. (#43384)" (#43647)"

This reverts commit e6d9009daff01f24b2207724d53f76bc8ba52d13.

sahandevs committed:
Revert "Ensure target platform is passed is always passed to assemble (#43862)"

This reverts commit 0e6881ed8f64907897ee534583c2a1252ffe9024.

sahandevs committed:
Revert "Reland: Migrate examples to Android embedding v2 (#44139)"

This reverts commit 07425f837ead4fce6f067a5cce1fe1c640039de5.

sahandevs committed:
Revert "Add enableFeedback param to MaterialButton, RawMaterialButton and IconButton (#41972)"

This reverts commit c8fbdaa7929b92a50ae0d557562941e0314009d1.

sahandevs committed:
Revert "Roll engine 5051bef17bdc..e609577d1201 (3 commits) (#43854)"

This reverts commit d4f630552c00cf801c5d4785e5c34cfaca53c118.

sahandevs committed:
Revert "Fixing focus traversal when the node options are empty (#43238)"

This reverts commit 532eb9d86123d21148b7ccb5905c2830b9409e23.

sahandevs committed:
Revert "remove listDartSources (#43950)"

This reverts commit a868d6413aa3bfb597489b14ca04b7b2344f36d6.

sahandevs committed:
Revert "Roll engine 6c763bb551cb..9726b4cb99d3 (4 commits) (#44251)"

This reverts commit 084592e319c008a21078b85328eca8011295cbc1.

sahandevs committed:
Revert "Roll engine b4a50d7ab192..e0b9a2bb23e4 (2 commits) (#44501)"

This reverts commit 99211430d73359350cd8cf24f70a26397714858f.

sahandevs committed:
Revert "check for instanceof instead of runtimeType and wait for testbed (#43870)"

This reverts commit f313d763ff9a2290fd38c63f93fb17f18af45622.

sahandevs committed:
Revert "Update create.dart (#43764)"

This reverts commit cffac34c67f1c9d8a2878984a9337fd96f378eb0.

sahandevs committed:
Revert "check if libimobiledevice executables exist (#43767)"

This reverts commit 3c9f20a010b86bc6db3918247c896d7929eb4da3.

sahandevs committed:
Revert "Roll engine c3b63d610b07..49971e21402e (3 commits) (#43673)"

This reverts commit 88bcaae212e591c9a35453b4b031983b665d54b0.

sahandevs committed:
Revert "Changing RenderEditable.textAlign doesn't break hot reload anymore (#44605)"

This reverts commit d9d9099cf4dac3ff28b3035cbb96f2becb3e4bd5.

sahandevs committed:
Revert "CupertinoTheme & CupertinoTextTheme dark mode updates (#41859)"

This reverts commit 9c66385d5f172082c7f84976443a72dde53576e9.

sahandevs committed:
Revert "Move the plugin registrant to io.flutter.plugins and add the @Keep annotation (#44011)"

This reverts commit 9242725f7617d8733c2aae9b0b5b20091ea43c27.

sahandevs committed:
Revert "6c257df8e Roll src/third_party/dart 6354b0b97d..d166b0be1c (4 commits) (#43563)"

This reverts commit 3934dc63eb005564f9a544feb42179c8b706123b.

sahandevs committed:
Revert "Make disposing a ScrollPosition with pixels=null legal (#44617)"

This reverts commit 3bca73b9a44f72f24af7e7c9901ef36de556f0cf.

sahandevs committed:
Revert "catch failure to parse FLUTTER_STORAGE_BASE_URL (#43599)"

This reverts commit fa1cfcf103f81e7c61afde757fbcef57de1e5452.

sahandevs committed:
Revert "Ensure Chrome is closed on tab close (#43586)"

This reverts commit b3e7ce1cbbb47b878d8e0a6d3ea1c80f2c7be5d8.

sahandevs committed:
Revert "Add reloadMethod RPC (#43725)"

This reverts commit a2b2a1cba281912e654cc18daca0999368028c68.

sahandevs committed:
Revert "Segmented control quick double tap fix (#44391)"

This reverts commit 391651196e09faa29476652ce7101a110e2c8121.

sahandevs committed:
Revert "disable chrome devicelab test (#44746)"

This reverts commit 9981db92714f31839871757976dc56a962fe26e9.

sahandevs committed:
Revert "Add macOS fn key support. (#44410)"

This reverts commit 7a57f4d31b6b2596adc7d64acb2fe3ca23878a7f.

sahandevs committed:
Revert "Revert "Revert "Reland: Migrate examples new embedding (#43868)" (#43885)" (#43939)"

This reverts commit 962010992694e73d3310c17af461fde130b8856a.

sahandevs committed:
Revert "enable avoid_web_libraries_in_flutter (#43739)"

This reverts commit 0a3902f887374c04e4af673fc813bc507c62aa61.

sahandevs committed:
Revert "Revert: Migrate examples to the new embedding (#43689)"

This reverts commit 55cd3875265966ddd7161d7521d58d95d29ed2bb.

sahandevs committed:
Revert "Don't allow Disabled InkWells to be focusable (#43848)"

This reverts commit 8e56bc9cd8c9db0cc7bcf4cec204a73db36250e0.

sahandevs committed:
Revert "Asset server fix for sourcemaps (#44017)"

This reverts commit 1e2d3a7640bc63e44414e63ec2cc4061fa905262.

sahandevs committed:
Revert "73541c97b [recipe] Upload sky_engine to CIPD (flutter/engine#13464) (#43933)"

This reverts commit d4ac18f81f9de2fab41e1acc575f044f9eacc109.

sahandevs committed:
Revert "Make DropdownButton's disabledHint and hint behavior consistent (#42479)"

This reverts commit f66a0f62ed3a3d72058b67c6f2cd9f4ff7baed2f.

sahandevs committed:
Revert "Add version to fuchsia_remote_debug_protocol (#44151)"

This reverts commit 0db420d92b2abe4943cba31dd856aa786cf37a66.

sahandevs committed:
Revert "Revert "[Gallery] Add Material Study app Rally as an example app (#42236)" (#44194)"

This reverts commit 14af8f3f026c3161e3529a5ea6c7f9d5f68c69c4.

sahandevs committed:
Revert "Release startup lock during long-lived build ios framework (#44625)"

This reverts commit 7edc16f90b6cee05469113b8bc01b94db98c0fe1.

sahandevs committed:
Revert "[Gallery] Add Material Study app Rally as an example app (#42236)"

This reverts commit e1b7d555329955b09dc35318a845801d4715b9d4.

sahandevs committed:
Revert "[flutter_tool] Teach the tool about local engine Fuchsia artifacts (#43759)"

This reverts commit 7033c4c7c3d2ca53d2513f183548fccdb1131ee7.

sahandevs committed:
Revert "Roll engine 7ed9c9fc5885..6a1f093398c6 (5 commits) (#44419)"

This reverts commit 8c4224329d3b4a16cadbac184634fc8a10eb48ed.

sahandevs committed:
Revert "Add v1 plugin register function into v2 plugin template (#44166)"

This reverts commit 07c0c8dcf60b0ee1059184792e51d82cb5c4e8c9.

sahandevs committed:
Revert "Fixing local golden output flake (#44307)"

This reverts commit 5a8d42d62d88c4f934764dbce05073b393e1f2d8.

sahandevs committed:
Revert "Fixed bug where we could accidently call a callback twice. (#43467)"

This reverts commit 062c108ef53250ca294ca9793522aeea35dc02cf.

sahandevs committed:
Merge pull request #3 from sahandevs/revert-2-master

Revert "Merge master"

sahandevs commented:
I Created a new pull request because of the mess https://github.com/flutter/flutter/pull/44971
